translations/strings.yaml

```diff
@@ -63,3 +63,21 @@ settingsDev:
   useStagingTitle: Use Staging Servers
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
+
+splittunnel:
+  missingAppActionButton:
+    value:
+      - Add apps manually
+    comment:
+      - Displayed in a notification as "call to action" when apps are missing from the split-tunnel settings.
+  missingAppMultiple: 
+    value:
+      - \%0 apps missing from list.
+    comment:
+      - Apps listed in the split-tunnel settings. %0 is the amount of apps missing
+  missingAppAddedOne: 
+    value:
+      - Success! You added %0 to this list.
```

Comment:
Maybe worth putting the name between quotes? Imagine the name of the app is "Send to"…

```
- Success! You added “%0” to this list.
```